scripts/abbreviations/main.swift

```diff
@@ -0,0 +1,149 @@
+#!/usr/bin/env xcrun swift -F ./Carthage/Build/Mac
```

Comment:
json2plist is neat, DOTADIW and probably won't need any changes so I wouldn't port it just yet.
I'd prefer to keep the abbreviation script in Swift for the sake of readability. We can probably cut a few lines and simplify but adding a plist and csv dependency.